src/pybind/mgr/cephadm/module.py

```diff
@@ -1499,7 +1634,7 @@ def _remove_daemon(self, name, host):
         """
         out, err, code = self._run_cephadm(
             host, name, 'rm-daemon',
-            ['--name', name])
+            ['--name', name, '--force'])
```

Comment:
when do we need force? Might make sense to be able to pass this from the cli

Reply:
We need `--force` for `rm-daemon` calls that operate on `mon|osd`. That's a constraint we get from `cephadm`.